### PR TITLE
Export isDOMElement in parts/Utilities.js

### PR DESCRIPTION
**Pull request by jon-a-nygaard**
Title says it all.
**NB!:** Dependent upon #11472, which should be merged first.

### CODE REVIEW

File: ts/parts/Utilities.ts [hunk 1]
```diff
@@ -1283,7 +1283,7 @@ function isObject(obj: any, strict?: boolean): boolean {
  * @return {boolean}
  *         True if the argument is a HTML Element.
  */
-H.isDOMElement = function (obj: any): boolean {
+function isDOMElement(obj: any): boolean {
```
Comment (bre1470) [inline ts/parts/Utilities.ts]: Did you tried `function isDOMElement(obj: unknown): obj is HTMLElement`?

Comment (jon-a-nygaard) [reply ts/parts/Utilities.ts]: No, will check that out 👍

Comment (jon-a-nygaard) [reply ts/parts/Utilities.ts]: @bre1470 Please see latest changes

Comment (bre1470) [reply ts/parts/Utilities.ts]: Looks great! Thanks. 👍

**Pull request merged**